pkg/products/threescale/reconciler.go

```diff
@@ -52,7 +52,8 @@ import (
 	userHelper "github.com/integr8ly/integreatly-operator/pkg/resources/user"
 
 	threescalev1 "github.com/3scale/3scale-operator/apis/apps/v1alpha1"
-	threescaleAmp "github.com/3scale/3scale-operator/pkg/3scale/amp/component"
+	//threescaleAmp "github.com/3scale/3scale-operator/pkg/3scale/amp/component"
```

Comment:
@valerymo Can this commented-out import for `threescaleAmp` be removed?

Reply:
Done. Thank you for review @carlkyrillos